test/browser_tests/cucumber/step_definitions/feedback-form.js

```diff
@@ -0,0 +1,90 @@
+const { Then, When, Before } = require( 'cucumber' );
+const chai = require( 'chai' );
+const expect = chai.expect;
```

Comment:
TRIVIAL: Probably could get rid of line 2 by using an import here `import { expect } from 'chai'`